x/ibc/03-connection/keeper/verify.go

```diff
@@ -32,7 +32,7 @@ func (k Keeper) VerifyClientConsensusState(
 	}
 
 	return clientState.VerifyClientConsensusState(
-		k.cdc, targetConsState.GetRoot(), height, connection.GetCounterparty().GetClientID(), consensusHeight, connection.GetCounterparty().GetPrefix(), proof, consensusState,
+		k.aminoCdc, targetConsState.GetRoot(), height, connection.GetCounterparty().GetClientID(), consensusHeight, connection.GetCounterparty().GetPrefix(), proof, consensusState,
```

Comment:
As long as `consensusState ` is proto, we don't need an amino codec if it's a hybrid codec. If it's not proto-defined, then yeah we'll have to use Amino for now. If it's not proto-defined, can it be?

Reply:
yeah, unfortunately I need the amino codec. It can't be proto defined because it'd break master if I add the tendermint changes.

Reply:
I see.